packages/ui-toolkit/stories/Toaster.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+
+import { StoryFn } from '@storybook/react';
+
+import { Toaster, toast, Button } from '../src/components/atoms';
+import { createPortal } from 'react-dom';
+
+export default {
+  title: 'Toaster',
+  component: toast,
+  tags: [ 'autodocs' ]
+};
+
+
+const Template: StoryFn = (_args) => {
```

Comment:
Can we add props as controls for Toaster stories ?

Reply:
done